examples/test_odx/test_server_odx.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import time
+import typing
+import unittest
```

Comment:
Same remark here, why is unittest needed?

Reply:
cleaned up